File: PegasusV1/PegasusV1/Entities/Evento.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace PegasusV1.Entities
{
    public class Evento
    {
        public int? Id { get; set; }
        public string? Nombre { get; set; }
        public string? Descripcion { get; set; }
        public DateTime? Fecha { get; set; }
        [Column("Requiere_Confirmacion")]
        public bool RequiereConfirmacion { get; set; }
        [Column("TIPO_DESTINATARIO")]
        public TipoDestinatario TipoDestinatario { get; set; }
    }
}

File: pegasusapp/src/models/evento.ts
export enum TipoDestinatario {
  SoloAlumnos = 1,
  SoloPadres = 2,
  Ambos = 3,
}

export interface Evento {
  id?: number | null
  nombre?: string | null
  descripcion?: string | null
  fecha: string
  requiereConfirmacion?: boolean
  tipoDestinatario?: TipoDestinatario
}
